abs.yaml

```diff
@@ -0,0 +1,2 @@
+channels:
+  - https://staging.continuum.io/prefect/fs/ephem-feedstock/pr6/3ba0d3a
```

Comment:
```suggestion
upload_channels:
  - sfe1ed40

channels:
  - https://staging.continuum.io/prefect/fs/ephem-feedstock/pr6/3ba0d3a
```
@Cansisti Sorry, I mistakenly removed the `upload_channel`

Reply:
I guess we should merge ephem 4.1.5 first anyway?

Reply:
No, it will go in the review channel as well. I'll prepare it